runtime/doc/repeat.txt

```diff
@@ -231,6 +231,10 @@ For writing a Vim script, see chapter 41 of the user manual |usr_41.txt|.
 				:runtime plugin/**/*.vim
 <			would source the first file only.
 
+			For each {file} pattern, if a file has both a `.vim`
+			and `.lua` extensions, the `.vim` version will take
+			priority and be sourced first.
```

Comment:
```suggestion
			For each {file} pattern, if both `.vim` and `.lua`
			files match in a directory, the `.vim` files take
			priority and are sourced first.
```

Reply:
This is stronger than what is intended. We just want to enforce that `foo/bar.vim` take priority over `foo/bar.lua`.

Reply:
At least add the "in a directory"

Reply:
How about this
```suggestion
			For each {file} pattern, if a multiple matches differ
			only in extension, the `.vim` version is sourced first
			the `.lua` version second, and others last.
```